octopoes/bits/oois_in_headers/oois_in_headers.py

```diff
@@ -16,12 +16,31 @@ def is_url(input_str):
     return bool(result.scheme)
 
 
+def get_ignored_url_params(config, config_key, default):
+    ignored_url_params = config.get(config_key, None)
+    if ignored_url_params is None:
+        return default
+    return [param.strip() for param in ignored_url_params.split(",")] if ignored_url_params else []
+
+
+def remove_ignored_params(url, ignored_params):
+    parsed_url = urlparse(url)
+    query_params = parse_qs(parsed_url.query)
```

Comment:
One potential improvement could be to handle cases where the URL might not have a query string or the query string is empty

```suggestion
    query_params = parse_qs(parsed_url.query)
    if not query_params:
        return url
```